docs/concepts/jcloud/yaml-spec.md

```diff
@@ -33,120 +31,94 @@ Since each Executor has its own business logic, it may require different cloud r
-```
+By default, C1 is allocated to each Executor and Gateway.
 
-#### Dedicated GPU
+JCloud offers the general Intel Xeon processor (Skylake 8175M or Cascade Lake 8259CL) for the CPU instances. 
 
-Using a dedicated GPU is the default way to provision GPU for an Executor. This automatically creates nodes or assigns the Executor to land on a GPU node. In this case, the Executor owns the whole GPU. You can assign between 1 and 4 GPUs.
+#### GPU Tiers
 
-```{code-block} yaml
----
-emphasize-lines: 5-7
----
-jtype: Flow
-executors:
-  - name: executor1
-    uses: jinaai+docker://<username>/Executor1
-    jcloud:
-      resources:
-        gpu: 1
-```
-
-### Spot vs on-demand instance
+JCloud supports GPU workloads with two different usages: `shared` or `dedicated`. 
 
-For cost optimization, JCloud tries to deploy all Executors on `spot` capacity. This is ideal for stateless Executors, which can withstand interruptions and restarts. It is recommended to use `on-demand` capacity for stateful Executors (e.g. indexers) however.
+If GPU is enabled, JCloud will provide NVIDIA A10G Tensor Core GPUs with 24G memory for workloads in both usage types.
 
-```{code-block} yaml
----
-emphasize-lines: 5-7
----
-jtype: Flow
-executors:
-  - name: executor1
-    uses: jinaai+docker://<username>/Executor1
-    jcloud:
-      resources:
-        capacity: on-demand
+```{hint}
+When using GPU resources, it may take a few extra minutes before all Executors are ready to serve traffic.
 ```
 
-### Memory
+| Instance | GPU    | Memory   | Credits Per Hour |
+|----------|--------|----------|------------------|
+| G1       | shared | 14 GB    | 100              |
+| G2       | 1      | 14 GB    | 125              |
+| G3       | 2      | 24 GB    | 250              |
+| G4       | 4      | 56 GB    | 500              |
 
-By default, `100M` of RAM is allocated to each Executor. You can use the `memory` argument under `resources` to change it.
+##### Shared GPU
 
-```{hint}
-Maximum of 16G RAM is allowed per Executor.
-```
+An Executor using a `shared` GPU shares this GPU with up to four other Executors.
+This enables time-slicing, which allows workloads that land on oversubscribed GPUs to interleave with one another.
 
-```{code-block} yaml
----
-emphasize-lines: 5-7
----
-jtype: Flow
-executors:
-  - name: executor1
-    uses: jinaai+docker://<username>/Executor1
-    jcloud:
-      resources:
-        memory: 8G
-```
+The tradeoffs with a `shared` GPU are increased latency, jitter, and potential out-of-memory (OOM) conditions when many different applications are time-slicing on the GPU. If your application is memory consuming, we suggest using a dedicated GPU.
+
+To use `shared` GPU, `G1` needs to be specified as instance type.
```

Comment:
```suggestion
```
move sentence up. It is now between two blocks that talk about dedicated GPUs

Reply:
done